File: responses/GPAUWQ-R8E/find_maximum_product.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(X, B):
    if len(B) == 0 or X > len(B):
        return 0
    else:
        out = 1
        for i in B[-X:]:
            out *= i
        return out

File: responses/MMVQP9-WAB/find_maximum_product.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
    var terms = A.length;
    var match = 1;
    var prefix = "";
    while (match == 1) {
        for (var c = 0; c < 100; c++) {
            for (var i = 0; i < terms-1; i++) {
                if (A[i][c] != A[i+1][c]) {
                    match = 0 
                }
            }            
        }
    }
    return prefix;
}